extensions/amp-story/1.0/amp-story-access.css

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-access {
+  position: absolute !important;
+  display: flex !important;
+  flex-direction: column !important;
+  top: 0 !important;
+  left: 0 !important;
+  height: 100% !important;
+  width: 100% !important;
+  z-index: 100003 !important;
+  transform: translate3d(0, 100vh, 0) !important;
+  transition-delay: 0.15s !important;
+}
+
+amp-story-access.i-amphtml-story-access-visible {
+  transform: translate3d(0, 0, 0) !important;
+  transition-delay: 0s !important;
+}
+
+/** Black opacity overlay. */
+amp-story-access::before {
+  content: "" !important;
```

Comment:
Just curious... why do we set this?

Reply:
By default, `::before` and `::after` pseudo elements have no content and thus are not generated. Setting an empty content ensures they are actually generated, so we can style and display them.

See: https://www.w3.org/TR/CSS2/generate.html#content